File: integrations/aws-strands/error-code-corpus/python/plain_literal.py
"""A terminal frame whose code and message are both written at the site."""

from ag_ui.core import EventType, RunErrorEvent


def emit() -> RunErrorEvent:
    return RunErrorEvent(
        type=EventType.RUN_ERROR,
        message="A plain literal message.",
        code="PLAIN_LITERAL",
    )

File: integrations/aws-strands/error-code-corpus/typescript/plain_literal.ts
/** A terminal frame whose code and message are both written at the site. */
import { EventType, type BaseEvent } from "@ag-ui/core";

export function emit(): BaseEvent {
  return {
    type: EventType.RUN_ERROR,
    message: "A plain literal message.",
    code: "PLAIN_LITERAL",
  };
}
